site/realm/functions/logReportHistory.js

```diff
@@ -0,0 +1,23 @@
+
+/**
+ * This function creates a entry into the Report History collection
+ * 
+ * @param {object} input A JSON object that contains the Report data
+ * 
+*/
+exports = async (input) => {
+
+  const reportsHistory = context.services.get('mongodb-atlas').db('history').collection("reports");
+
+  const report = input;
```

Comment:
The payload defined below only asks for `repor_number`, but the function here assumes that the entire report is being sent 🤔

Reply:
The payload is the returned value, the Input type is: `"input_type": "History_reportInsertInput",`